src/verifiable/credential.ts

```diff
@@ -134,6 +137,128 @@ export class W3CCredential {
     return undefined;
   }
 
+  /**
+   * Verify credential proof
+   *
+   * @returns {*}  {(boolean)}
+   */
+  async verifyProof(
+    proofType: ProofType,
+    resolverURL: string,
+    credStatusResolverRegistry?: CredentialStatusResolverRegistry
```

Comment:
I suspect in the future we will need some additional options. Can we add the interface W3CProofVerificationOptions  and as optional field credStatusResolverRegistry, make this options also optional

Reply:
added